src/main/java/com/google/googleidentity/oauth2/endpoint/TokenEndpoint.java

```diff
@@ -39,6 +41,8 @@
 import java.net.URLDecoder;
 import java.util.logging.Logger;
 
+import static com.google.googleidentity.oauth2.request.OAuth2Request.RequestBody.ResponseType;
```

Comment:
nit: static imports go before regular ones. But also, just use regular imports for static nested classes like this one.

https://google.github.io/styleguide/javaguide.html#s3.3-import-statements
https://google.github.io/styleguide/javaguide.html#s3.3.4-import-class-not-static